browser/ui/views/frame/brave_system_menu_model_builder.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_FRAME_BRAVE_SYSTEM_MENU_MODEL_BUILDER_H_
+#define BRAVE_BROWSER_UI_VIEWS_FRAME_BRAVE_SYSTEM_MENU_MODEL_BUILDER_H_
+
+#include "chrome/browser/ui/views/frame/system_menu_model_builder.h"
+
+// This class can be used to update toolbar/frame context menus.
+class BraveSystemMenuModelBuilder : public SystemMenuModelBuilder {
+ public:
+  BraveSystemMenuModelBuilder(ui::AcceleratorProvider* provider,
+                              Browser* browser);
```

Comment:
we could replace this with `using SystemMenuModelBuilder::SystemMenuModelBuilder;` as this ctor just call base class's ctor. and can delete ctor definition in cc.